leader-election-impl/src/main/java/com/palantir/leader/proxy/AwaitingLeadershipProxy.java

```diff
@@ -50,8 +58,12 @@
 
     private static final Logger log = LoggerFactory.getLogger(AwaitingLeadershipProxy.class);
 
-    private static final long MAX_NO_QUORUM_RETRIES = 10;
+    private static final int MAX_NO_QUORUM_RETRIES = 10;
```

Comment:
This class should probably do with some better unit tests, now, before this merges.